elasticdl/python/worker/task_data_service.py

```diff
@@ -94,8 +96,8 @@ def report_record_done(self, count, err_msg=""):
                     self._reported_record_count -= task.end - task.start
                     self._pending_tasks.popleft()
                     self._do_report_task(task, err_msg)
-                    # Becuase a single batch comes from multiple tasks,
-                    # We just report the number of failing records together
+                    # Since a single batch comes from multiple tasks,
```

Comment:
IMO, in current implementation, a single batch only comes from a single task

Reply:
Agreed. Just removed this comment. The purpose of this PR was only for improving code style and fixing English grammar.